### ISSUE TITLE
Prefetch error for dynamic route

### ISSUE THREAD

**Issue by e16m** [fireship-io/next13-pocketbase-demo#16]
I'm seeing an issue at this point in the tutorial:

```
function Note({ note }: any) {
  const { id, title, content, created } = note || {};

  return (
    **<Link href={`/notes/${id}`}>**
      <div className={styles.note}>
        <h2>{title}</h2>
        <h5>{content}</h5>
        <p>{created}</p>
      </div>
    </Link>
  );
}
```

When I change it to "/" or "/notes" it works fine, but when I try to do this I get the following error:

**PREFETCH ERROR Error: Functions cannot be passed directly to Client Components because they're not serializable.**

Does anyone know how to handle this error? I have been researching this for the past few hours but no luck so far.. thanks.

**Comment by suti1979** [comment]
same here... :(

**Comment by jameshd** [comment]
Yep same here. I found deleting the error.tsx got rid of it... not sure why it happens though...

**Comment by jameshd** [comment]
https://github.com/fireship-io/next13-pocketbase-demo/pull/3 this fixes it... basically `'use client';`

**Comment by e16m** [comment]
@jameshd thanks for sharing!
